heli/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "heli",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://helichain.com/",
+  "pretty_name": "HeliChain",
+  "chain_id": "helichain",
+  "bech32_prefix": "heli",
+  "daemon_name": "helichaind",
+  "node_home": "$HOME/.helichain",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uheli",
+        "fixed_min_gas_price": 0.001,
+        "low_gas_price": 0.001,
+        "average_gas_price": 0.002,
+        "high_gas_price": 0.0025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uheli"
+      }
+    ],
+    "lock_duration": {
+      "time": "1209600s"
+    }
+  },
+  "codebase": {
+    "git_repo": "https://github.com/Heli-Chain/heli",
+    "recommended_version": "v0.0.1",
+    "compatible_versions": [
+      "v0.0.1"
+    ],
+    "cosmos_sdk_version": "0.47.4",
+    "ibc_go_version": "v7.2.0",
+    "consensus": {
+      "type": "cometbft",
+      "version": "0.37.2"
+    },
+    "cosmwasm_version": "0.41.0",
+    "cosmwasm_enabled": true,
+    "genesis": {
+      "genesis_url": "https://raw.githubusercontent.com/Heli-Chain/heli/main/genesis.json"
+    },
+    "versions": [
+      {
+        "name": "v0.0.1",
+        "recommended_version": "v0.0.1",
+        "compatible_versions": [
+          "v0.0.1"
+        ],
+        "cosmos_sdk_version": "0.47.4",
+        "ibc_go_version": "v7.2.0",
+        "consensus": {
+          "type": "cometbft",
+          "version": "0.37.2"
+        },
+        "cosmwasm_version": "0.42.0",
+        "cosmwasm_enabled": true,
+        "next_version_name": ""
+      }
+    ]
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "8f86b9543ee84a69072cb8347cc8fb4ace68e6f5",
+        "address": "45.77.172.45:26656"
+      },
+      {
+        "id": "60d5b2568bc9d6e0a61e4f0ab2ed3ace71b8dcb1",
+        "address": "66.42.53.120:26656"
+      },
+      {
+        "id": "c1bc48f2bf4361d2c4afd135df8e4b0020672877",
+        "address": "149.28.153.206:26656"
+      }
+    ],
+    "persistent_peers": [
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://rpc.helichain.com/",
+        "provider": "HeliChain"
+      }
+    ],
+    "rest": [
+      {
+        "address": "https://lcd.helichain.com/",
+        "provider": "HeliChain"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "ping.pub",
+      "url": "https://explorer.helichain.com/Helichain",
+      "tx_page": "https://explorer.helichain.com/Helichain/tx/${txHash}",
+      "account_page": "https://explorer.helichain.com/Helichain/account/${accountAddress}"
+    }
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/heli/images/heli.png",
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/heli/images/heli.svg"
```

Comment:
The png and svg you have here are not identical. One has transparect background, while the other has a white background. Please make sure they are the same, or else register them as separate images.